tests/manage/mcg/test_bucket_policy.py

```diff
@@ -370,15 +371,25 @@ def test_object_actions(self, mcg_obj, bucket_factory):
         logger.info(
             f"Getting object on bucket: {obc_obj.bucket_name} with user: {user.email_id}"
         )
-        assert s3_get_object(
-            user, obc_obj.bucket_name, object_key
-        ), "Failed: Get Object"
+        for get_resp in TimeoutSampler(
+            60, 1, s3_get_object, user, obc_obj.bucket_name, object_key
```

Comment:
```suggestion
            30, 4, s3_get_object, user, obc_obj.bucket_name, object_key
```
I think it might be better to prolong the time in which we try to check this, and to shorten the time between checks - this way we'll have better coverage for both quicker and slower endpoint syncs